assets/scss/_styles_project.scss

```
@@ -987,52 +987,30 @@ td > ul, td > ol {
   }
 }
 
+// relatedcard styling
 
-// relatedlist styling
-
-ul.relatedlist {
-  padding: 0;
+.relatedcard {
   display: flex;
+  flex-direction: column;
 }
 
-ul.relatedlist > li {
-  list-style-type: none;
-  padding-right: 2rem;
-}
-
-ul.relatedlist > li > a {
+.relatedcard a {
+  width: 7rem;
   text-decoration: none;
-}
-
-ul.relatedlist > li > a > div {
-  width: 100%;
-  padding: 0.25rem 0rem;
-  align-items: center;
+  padding: 1.25rem;
```

I think slightly less padding works too - and that deals with the data management one
```suggestion
  padding: 0.75rem;
```